Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -482,6 +501,22 @@ final class ChatViewController: SLKTextViewController {
         return true
     }
 
+    private func observeAlertSubscriptions() {
+        guard let auth = AuthManager.isAuthenticated() else { return }
+        let subscriptions = auth.subscriptions
+
+        alertSubscriptionToken = subscriptions.addNotificationBlock { [weak self] _ in
```

Comment:
I think the token will be destroyed along with the key window reassignment on user's logout.
I'll add the de-registration though.